src/Notifiers/Slack/SlackNotifier.php

```diff
@@ -239,9 +239,13 @@ private function buildCfpsBlocks(array $endingCfps): array
     /** @param array<mixed> $blocks */
     private function notify(array $blocks): void
     {
-        $this->httpClient->request('POST', $this->webHookUrl, [
-            'headers' => ['Content-type' => 'application/json'],
-            'body' => json_encode($blocks),
-        ]);
+        try {
+            $this->httpClient->request('POST', $this->webHookUrl, [
+                'headers' => ['Content-type' => 'application/json'],
+                'body' => json_encode($blocks),
+            ]);
+        } catch (\Exception $exception) {
```

Comment:
**Never** use \Exception (nor \Throwable). Everything because harder to debug with that.